action-ssh-to-runner/action.yaml

```diff
@@ -14,6 +14,10 @@ inputs:
   tmate-server-ed25519-fingerprint:
     description: Tmate server ed25510 key fingerprint
     required: true
+  detached:
+    description: 'Run tmate in detached mode ("true" or "false", default: "false").'
+    required: false
+    default: "false"
```

Comment:
Can you check if we can use proper bools here? 


```suggestion
  detached:
    description: 'Run tmate in detached mode (true or false, default: false).'
    required: false
    default: false
```

Reply:
I tried and it said github doesn't support it yet.

Reply:
<img width="467" alt="Screenshot 2024-11-08 at 16 28 22" src="https://github.com/user-attachments/assets/36330e9b-92fb-48c7-8c3d-e517ebe2c0ba">

Reply:
<img width="652" alt="Screenshot 2024-11-08 at 16 29 24" src="https://github.com/user-attachments/assets/34307d8c-a199-4ada-84bc-46e3dd2a4d39">

Reply:
What if you specified the input type `type: boolean` ?

Reply:
I did [here](https://github.com/scality/actions/pull/154#discussion_r1834584673)
the first screenshot

Reply:
thanks for trying!